nemo/lightning/pytorch/callbacks/peft.py

```diff
@@ -126,9 +128,11 @@ def freeze_model(self, model: nn.Module) -> None:
             model.module.freeze()
         else:
             model.freeze()
-        model.train(mode=True)
+        if hasattr(model, "trainer") and model.trainer.state == TrainerFn.FITTING:
```

Comment:
Should use model.trainer.state.fn != TrainerFn.FITTING instead.

Reply:
ah thanks!